ebuild-writing/variables/text.xml

```diff
@@ -153,6 +153,33 @@ for these variables when writing ebuilds.
 		satisfied by <c>BDEPEND</c>, typically executable build tools.
     </ti>
   </tr>
+  <tr>
+    <ti><c>MERGE_TYPE</c></ti>
+    <ti>
+      The type of package that is being merged. Possible values are:
+      <c>source</c> if building and installing a package from source,
+      <c>binary</c> if installing a binary package,
```

Comment:
Should we further clarify what we mean by a binary package? They apply to Gentoo-specific binary packages that are built using a PMS-compliant package manager. If you are merging `firefox-bin`, the merge type would still be `source` unless there is a binary package for `firefox-bin` is available somewhere. Thoughts?

Reply:
We should, but this table isn't the right place for it. Presumably, we should have a dedicated section for binary packages (and there could be a link to it in the table).

Also, this text was there before, and I've only moved it to a new place.

Reply:
I propose as a quick fix: s/a binary package/a Gentoo binary package/. What do you think?

Reply:
"a previously built package"?

Reply:
How about:

```suggestion
      <c>binary</c> if installing a binary package built from the ebuild,
```

that should make it clear what kind of binary package we're talking about. I don't think it's a good idea to try to find a roundabout name for it, as that will only confuse people.

Reply:
Even better, but I'd still say "**previously** built from the ebuild" to make it clear that we're not building it as part of this cycle.